src/components/forms/metadata-handlers/familyForm.ts

```diff
@@ -55,11 +55,22 @@ interface IFamilyFormAFProjects extends IFamilyFormMcfProjects {
   sector?: IChakraSelect[]
 }
 
+interface IFamilyFormGcfProjects extends IFamilyFormMcfProjects {
+  sector?: IChakraSelect[]
+  result_area?: IChakraSelect[]
+  result_type?: IChakraSelect[]
+  theme?: IChakraSelect[]
+  approved_ref?: string
```

Comment:
I think i forgot to write this in my previous review, why are the values here potentially undefined `?:` surely we would always expect these in the GCF data

Reply:
I've reminded myself, sorry for the late reply! This isn't to do with the data we're retrieving from the API (which, you're right - we'd always expect values for these properties). This is instead to do with the family form itself, where properties either exist or are undefined (think of the `reset` function) :)